README.md

```diff
@@ -118,6 +118,15 @@ Most types of add-on content can be downloaded within OpenTTD via the 'Check Onl
 Add-on content can also be installed manually, but that's more complicated; the [OpenTTD wiki](https://wiki.openttd.org/) may offer help with that, or the [OpenTTD directory structure guide](./docs/directory_structure.md).
 
 
+### 1.5.1) Social Integration
```

Comment:
If I'm a privacy-conscious user downloading the game from Steam (cherry-picking when I care about my privacy, naturally 😛), I get the plugins installed automatically. It would be nice to have a sentence here telling me that if I delete the file, the plugin is gone. Just to sooth worried minds.

Reply:
I am not sure.

Expecting Steam integration to be shipped with a game on Steam is kinda a no-brainer. Nobody is going to expect it to not be there, as most of the games on Steam ship it. If you are privacy conscious .. you would not have used Steam to download the game :D

For the Discord plugin on Steam, there you might have a point. I could make it a Steam DLC, if that helps.

The problem with deleting is that Steam will just restore it for you. So there is little point in suggesting something that is not going to work :)

Reply:
Hmm, DLC could be a good way to distribute it. I agree that shipping Steam integration with a Steam build shouldn't bother anybody -- and it really does, they can download through openttd.org.